charts/argo-cd/templates/argocd-application-controller/deployment.yaml

```diff
@@ -86,6 +86,11 @@ spec:
         - name: controller
           containerPort: {{ .Values.controller.containerPort }}
           protocol: TCP
+        {{ if .Values.controller.metrics.enabled }}
+        - name: metrics
+          containerPort: 8083
```

Comment:
Is this correct port?

Looking at `.Values.controller.metrics.service.servicePort` it's set to 8082 :thinking: 

8083 seems to be used for `server` metrics.

Reply:
Yes it is the correct port, looks like the ports were changed at some point, the argocd docs, values.yaml, and the metrics service template may be outdated.

If you compare with the API server and repo server, it looks like 8083 is the new metrics port. Can confirm that I got the API server and application-controller working on 8083/metrics/